.github/workflows/main.yml

```diff
@@ -639,3 +652,32 @@ jobs:
       uses: geekyeggo/delete-artifact@v1
       with:
         name: Realm.UnityWeaver.${{ needs.build-packages.outputs.package_version }}.nupkg
+  generate-docs:
+    if: ${{ contains(github.ref, 'release') }}
+    runs-on: windows-latest
+    name: Generate docs
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+      with:
+        submodules: recursive
+    - name: Register csc problem matcher
+      run: echo "::add-matcher::.github/problem-matchers/csc.json"
+    - name: Register msvc problem matcher
+      run: echo "::add-matcher::.github/problem-matchers/msvc.json"
+    - name: Build docs
+      run: |
+        choco install docfx -yes
+        docfx Docs/docfx.json
+        $resultPath = "Realm/packages/"
+        if(-not(Test-Path -path $resultPath))
+        {
+            New-Item -ItemType directory -Path $resultPath
+        }
+        Compress-Archive -Path Docs/_site -DestinationPath "$resultPath/Docs.zip"
+    - name: Store docs artifacts
+      uses: actions/upload-artifact@v2
+      with:
+        name: Docs.zip
+        path: ${{ github.workspace }}/Realm/packages/Docs.zip
+        retention-days: ${{ github.event_name != 'pull_request' && 30 || 1 }}
```

Comment:
I'm not sure the amount of days we'd like to retain the docs. So I just put the standard amount length we always use. Please, advise if 30 days are too many.